docs/conf/prague/2018/news/call-for-volunteers.rst

```diff
@@ -0,0 +1,50 @@
+:template: {{year}}/generic.html
+
+.. post:: June 15, 2018
+   :tags: prague-2018, volunteers, tickets
+
+Ticket Update, Call for Volunteers, Job Fair
+============================================
+
+Greetings, fellow documentarians! The Portland conference is less than three months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+**Tickets are selling fast!** We are not sold out yet, but our European community is showing great enthusiasm, and we expect to have another wave of registrations after we announce the presentations soon, so you might want to beat the next rush and `get your ticket now <https://ti.to/writethedocs/write-the-docs-prague-2018>`_.
```

Comment:
s/soon/in a week or so/ maybe?

Reply:
also remove 'next' from rush?

Reply:
I'm not sure about "in a week" because that's more specific than "soon"...
Will remove "next".